pkg/metrics/platform_test.go

```diff
@@ -86,6 +87,17 @@ var _ = Describe("PlatformMetrics", func() {
 				Expect(result).To(Equal(expectedValue))
 			})
 
+			It("should increment provisioning_successes metric", func() {
+				HandleMetrics(platform, func(m *PlatformMetrics) {
+					rnd := rand.Intn(100)
+					expectedValue = rnd
+					m.ProvisionSuccesses.Add(float64(rnd))
+				})
+				result, err := getCounterValue(platform, provisionSuccessesMetricName)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(result).To(Equal(expectedValue))
```

Comment:
minor: this can be simplified as follows:

```suggestion
				Expect(getCounterValue(platform, provisionSuccessesMetricName)).To(Equal(expectedValue))
```